start_mcp_server.py

```
@@ -0,0 +1,62 @@
+logger = logging.getLogger(__name__)
+
+def start_mcp_server():
+    """Start the MCP server with proper error handling"""
+    
+    server_path = Path(__file__).parent / "mcp_server" / "main.py"
+    
+    try:
+        logger.info("Starting MCP server...")
+        
+        # Start server process
+        process = subprocess.Popen([
```

Ruff (0.11.9)

`subprocess` call: check for execution of untrusted input

(S603)